src/formatting/lists.rs

```diff
@@ -461,6 +461,13 @@ where
 
             let mut formatted_comment = rewrite_post_comment(&mut item_max_width)?;
 
+            // Mmultiline comments are not included in a previous "indentation group".
+            // Each multiline comment is a considered as a separage group.
```

Comment:
Minor typos

```suggestion
            // Multiline comments are not included in a previous "indentation group".
            // Each multiline comment is a considered as a separate group.
```

Reply:
Done (with removing the first "a" in the second line)